dids/src/main/kotlin/web5/sdk/dids/methods/jwk/DidJwk.kt

```diff
@@ -52,7 +54,7 @@ public object DidJwk {
     keyManager: KeyManager = InMemoryKeyManager(),
     algorithmId: AlgorithmId = AlgorithmId.Ed25519): BearerDid {
 
-    val keyAlias = keyManager.generatePrivateKey(algorithmId)
+    val keyAlias = keyManager.generatePrivateKey(algorithmId, DefaultKeyGenOptions("0"))
```

Comment:
implement key gen options so that we can define the key alias at generation time